memtable/write_buffer_manager.cc

```diff
@@ -78,6 +79,34 @@ WriteBufferManager::~WriteBufferManager() {
 #endif  // ROCKSDB_LITE
 }
 
+// Should only be called from write thread
+bool WriteBufferManager::ShouldFlush() const {
```

Comment:
Any particular reason why this is here instead of the header file? ```ShouldFlush()``` is called by every write, so keeping it in the header file may make it more likely to be inlined. Maybe compiler would optimize it anyway, but just to be on the safe side.